content/en/docs/duplicate-certificate-limit.md

```diff
@@ -23,6 +24,31 @@ set" of hostnames for your certificate would be `[example.com]`. This rate limit
 is exceeded when a subscriber requests a certificate for the same "exact set" of
 hostnames more than 5 times in a single week.
 
+## Wait for Reset
+
+Revoking the previously issued certificates will not reset the Duplicate
+Certificate limit. Subscribers can request another certificate 168 hours (7
+days) after the  most recent successful issuance. If you're unsure of the date
+and time of your last successfully issued certificate, follow the directions
+below:
+
+- Open [crt.sh](https://crt.sh/).
+- Input your domain name and click the **Search** button.
+- Click to open the detailed view for the top-most entry.
+- Note the **Not Before** timestamp under the **Validity** section.
+- At 168 hours from that timestamp you can request a new certificate.
+
+## Workaround
```

Comment:
I prefer workaround to be at the bottom in hopes readers review the common causes first and don't keep using the work around because they won't fix the root cause.